### PR TITLE
Improve docs around setting admin usernames in tap-values file

### PR DESCRIPTION
**Pull request by akrishna90**
- Rephrase the note about making sure the admin usernames refer to actual identities that can be used as users in Kubernetes RBAC resources
- Add more possible causes and troubleshooting steps for `Org Not Found` issue

[TME-2229]

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -327,23 +327,23 @@ Organization 'my-org' not found
 
 #### Possible causes
 
-The user creating the CFOrg is not bound to the `korifi-controllers-admin`
-ClusterRole.
+1. The user creating the CFOrg does not have a `RoleBinding` that is bound to the `korifi-controllers-admin` ClusterRole.
+2. The user has a `RoleBinding` but the binding is assigned to a username that is defined in the user's local Kubeconfig but isn't a recognized user on the Kubernetes cluster.
```
Comment (tcdowney) [inline troubleshooting.md:331]:
```suggestion
2. The user appears to have a `RoleBinding`, but the binding is assigned to a local Kubeconfig user entry that does not match a recognized user on the Kubernetes cluster.
```

Comment (tcdowney) [reply troubleshooting.md:331]: This one's hard to word. 😅 I suggested a tweak, but not sure myself if that is any clearer. Maybe @HenryBorys can help us out here. 🙂

File: install.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ VMware recommends configuring one or more administrative users for Application S
...
+    It is important to note that, these usernames must be identities which can be recognized as users in subject section of Kubernetes RBAC resources, such as `RoleBindings`. They can differ and might not match the user entries in your local Kubeconfig file. During the initial installation, you can leave this entry blank if you're unsure of the username. After installation, login with the cf CLI and run the `cf curl /whoami` command to confirm the username. Then, update the installation with the correct name value. For more information about user subject names in Kubernetes, see the [Referring to subjects](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-subjects) section of _Using RBAC Authorization_ and the [Authenticating](https://kubernetes.io/docs/reference/access-authn-authz/authentication/) topic in the Kubernetes project documentation.
```
Comment (tcdowney) [inline install.md:294]: I think this is quite a bit clearer!

File: troubleshooting.md (hunk 1 above)
Comment (HenryBorys) [reply troubleshooting.md:331]: Looks good. I just removed a couple of extra words.

**Review by tcdowney — APPROVED** [review]
LGTM

**Pull request merged**